### ISSUE TITLE
Windows Subsystem for Linux - Core Dumped

### ISSUE THREAD

**Issue by tylerwmarrs** [matrix-profile-foundation/matrixprofile#13]
**Describe the bug**
When running any Cython code in WSL it errors out with "Illegal instruction (core dumped)". This is due to major differences in the binary wheel of a **true** Linux versus WSL.

**To Reproduce**
1. install matrixprofile via pip on WSL
2. try to run the following
```python
import matrixprofile as mp

ds = mp.datasets.load('motifs-discords-small')
ts = ds['data']
profile = mp.compute(ts)
```

Result is an error:
```
Illegal instruction (core dumped)
```

**Expected behavior**
This should give the profile as expected.

**Desktop (please complete the following information):**
 - OS: Windows 10 64 bit
 - WSL: Ubuntu - 4.4.0-18362-Microsoft #476-Microsoft Fri Nov 01 16:53:00 PST 2019 x86_64 x86_64 x86_64 GNU/Linux

**Workaround**
Installing from source works fine.
```
sudo apt install build-essential
git clone git@github.com:matrix-profile-foundation/matrixprofile.git
cd matrixprofile
pip install -e .
```

**Comment by tylerwmarrs** [comment]
The documentation should be updated to reflect this issue and guide WSL users to install from source instead of relying on the binary installation via pip.

**Comment by tylerwmarrs** [comment]
After further investigation, the proper fix is to change the C compile flags.

**Comment by tylerwmarrs** [comment]
This commit resolves this issue - d7ee9530bf09e266d44098625c272e32e38155ec
